scripts/partition_manager.py

```diff
@@ -675,6 +675,11 @@ def get_dynamic_area_start_and_size(static_config, base, size, dp):
     return start, end - start
 
 
+def get_end_address(pm_config):
```

Comment:
This is not getting anything, as there is no return statement.

and why is this not part of `solve_simple_region()` and `solve_complex_region()` where the `pm_config[partition_name]['address'|'size'|'device']` is being set ?

Reply:
Function renamed to calculate_end_address.